Dockerfile

```diff
@@ -6,11 +6,10 @@ COPY package.json ./package.json
 COPY yarn.lock ./yarn.lock
 # copy `dist` if it exists already
 COPY dis[t]/web ./dist/
```

Comment:
```suggestion
COPY dis[t]/web ./dist/web
```

I apologise, because I believe I suggested this line before, but after testing again, I think it should actually be this.

![image](https://github.com/MarmadileManteater/FreeTubeCordova/assets/106682128/5e0e5d8f-324a-45e7-a9b8-9da7d68b26bb)